docs/source/refs/cli.md

```diff
@@ -200,6 +200,7 @@ __storage subcommands__
 | `upgrade 2to3`   | Upgrades legacy Aim repository from `2.x` to `3.0`.                                      |
 | `upgrade 3.11+`  | Update metric sequence data format for given runs. At least one run should be specified. |
 | `restore`        | Rollback `Run` to old metric format if run backup is available.                          |
+| `prune`          | Remove orphan params/sequences with no referring runs                                    |
```

Comment:
worth describing what we mean by saying "orphan"

Reply:
maybe even a small section in docs, describing a use-model (i.e. run moved/deleted, params overwritten, etc.)

Reply:
done ✅